gpkit/nomials/math.py

```diff
@@ -409,6 +366,8 @@ def __ge__(self, other):
     def mono_approximation(self, x0):
         return self
```

Comment:
Nope! Nomials are immutable, and Variables are only mutated directly after their creation (as in `bounded`)